lib/tokens.js

```diff
@@ -1059,9 +1079,7 @@ async function _getUnfilledBatch({
   // `tokenBatch` collection using `tokenBatch.id` shard key)
   const {batchInvalidationCount} = entityRecord.entity;
   const batchId = entityRecord.entity.openBatch[key];
-  const query = {'tokenBatch.id': batchId};
-  const projection = {_id: 0};
-  let record = await collection.findOne(query, {projection});
+  let record = await _getBatch({id: batchId});
```

Comment:
This breaks the code because it will throw if the `batchId` is not found. Previously, this did not call `_getBatch` (which throws on not found) to avoid that. So this will need to be fixed in some way to ensure it behaves the same way it used to.